go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/openshift/cluster-olm-operator
 
-go 1.23.0
+go 1.24.0
 
-toolchain go1.23.4
+toolchain go1.24.5
```

Comment:
OCP builds don't support go1.24.5, it supports go1.24.4.

Reply:
helm just requires [1.24.0](https://github.com/helm/helm/blob/98d6d9c1efeb7707dd331484645c48e1215baf05/go.mod#L3), so we should be able to do the same here, no?